### PR TITLE
Add Snapshot Tests for Call Visualizer

### PR DESCRIPTION
**Pull request by rasmustautsglia**
This PR covers snapshot tests for the call visualizer, including screen sharing, visitor code, and video call views. For better testability, mock objects were created or refactored

MOB-1777

### CODE REVIEW

File: GliaWidgetsTests/CallVisualizer/ScreenSharing/Mocks/ScreenSharingView.Mock.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        style: ScreenSharingViewStyle = .mock(),
+        header: Header.Props = .mock(),
+        endScreenSharing: ActionButton.Props = .mock()
+    ) -> CallVisualizer.ScreenSharingView.Props {
+        return .init(style: style, header: header, endScreenSharing: endScreenSharing)
+    }
+}
+
+#endif
+//endScreenSharing: .init(
```
Comment (gersonnoboa) [inline GliaWidgetsTests/CallVisualizer/ScreenSharing/Mocks/ScreenSharingView.Mock.swift:16]: this can be deleted

File: SnapshotTests/Changes.diff [hunk 2]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (gersonnoboa) [inline SnapshotTests/Changes.diff:1]: Is this file needed?

**Review by gersonnoboa — APPROVED** [review]

File: GliaWidgetsTests/CallVisualizer/ScreenSharing/Mocks/ScreenSharingView.Mock.swift (hunk 1 above)
Comment (rasmustautsglia) [reply GliaWidgetsTests/CallVisualizer/ScreenSharing/Mocks/ScreenSharingView.Mock.swift]: fixed [988c3dc](https://github.com/salemove/ios-sdk-widgets/pull/521/commits/988c3dc34073022cd6df7569db79d476a7a6646f)

File: SnapshotTests/Changes.diff (hunk 2 above)
Comment (rasmustautsglia) [reply SnapshotTests/Changes.diff]: fixed [988c3dc](https://github.com/salemove/ios-sdk-widgets/pull/521/commits/988c3dc34073022cd6df7569db79d476a7a6646f)

**Review by EgorovEI — APPROVED** [review]

**Review by yurii-glia — APPROVED** [review]

**Comment by yurii-glia** [comment]
!squash

**Review by igorkravchenko — APPROVED** [review]

**Comment by rasmustautsglia** [comment]
!merge
